lib/types/time-uuid.js

```diff
@@ -206,13 +206,14 @@ class TimeUuid extends Uuid {
      * @returns {{date: Date, ticks: Number}}
      */
     getDatePrecision() {
-        const timeLow = this.buffer.readUInt32BE(0);
+        const buffer = this.getBuffer();
+        const timeLow = buffer.readUInt32BE(0);
 
         let timeHigh = 0;
-        timeHigh |= (this.buffer[4] & 0xff) << 8;
-        timeHigh |= this.buffer[5] & 0xff;
-        timeHigh |= (this.buffer[6] & 0x0f) << 24;
-        timeHigh |= (this.buffer[7] & 0xff) << 16;
+        timeHigh |= (buffer[4] & 0xff) << 8;
+        timeHigh |= buffer[5] & 0xff;
+        timeHigh |= (buffer[6] & 0x0f) << 24;
+        timeHigh |= (buffer[7] & 0xff) << 16;
```

Comment:
Why do we need that? Why not implement a `get`ter for `buffer`, so that all reads from the field succeed?

Reply:
Okay, I forgot about this option. This means TimeUuid only needs to be changed to classes syntax to work. I applied this changes in [PR#60](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/pull/60) so I will close this one.